.github/ISSUE_TEMPLATE/1-bug.yaml

```diff
@@ -0,0 +1,83 @@
+name: "🐛 Plugin Bug Report"
+description: "Submit a plugin bug report to help us improve"
+title: "🐛 <Plugin>: <Title>"
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: We value your time and effort to submit this bug report. 🙏
+  - type: input
+    id: plugin-name
+    attributes:
+      label: Plugin Name
+      description: "Specify the name of the [plugin name](https://github.com/backstage/community-plugins/tree/main/workspaces) this issue refers to"
```

Comment:
this should show the list of all `@backstage-community/*` packages published on npm

```suggestion
      description: "Specify the name of the [plugin name](https://www.npmjs.com/search?q=%40backstage-community) this issue refers to"
```